### PR TITLE
Changelog!

### PR DESCRIPTION
**Pull request by evangael**
Clearly needed for your project!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+## [Unreleased]
+
+* Lots of stuff way to complicate for my drunk ass
```
Comment (jvautier) [inline CHANGELOG.md:6]: Can you be a little less aggressive here ?
There is other ways to say you are dumb you know ?

Comment (Bachib) [inline CHANGELOG.md:6]: too*

Comment (evangael) [reply CHANGELOG.md:6]: :'(

**Comment by evangael** [comment]
fix issue #4

**Review by Bachib — CHANGES_REQUESTED** [review]
Thanks for you contribution, few comments.

**Comment by evangael** [comment]
Will close this PR, since it's not up to the standard expected!

**Pull request closed without merging**